code/clement/html_css_flask/lab01/templates/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <link rel="stylesheet" href="{{url_for('static', filename='styles/index.css')}}">
+
+        <title>Bio-data</title>
+    </head>
+<body>
+
+    <section class="container" id="section-1"> 
+        <h1>Welcome To My First Bio Lab</h1>
+        
+        <section class="container" id="section-1">
+            <div class="card">
+                <div class="image-container">
+                    <img src="https://cdn1.edgedatg.com/aws/v2/abc/JudgeSteveHarvey/person/3894227/e674d10a10e0b080f744c70bbb1648f6/320x180-Q90_e674d10a10e0b080f744c70bbb1648f6.jpg" alt="My image">
+                    <h3 class="caption">Steve Harvey/Date of birth</h3>
+                    <h5 class="caption">January 17, 1957</h5>
+    
+                    <!-- A written introduction -->
+                    <p>
+                        Welch is a city located in McDowell County in the State of West Virginia, United States. The population was 3,590 at the 2020 census. Incorporated as a city in 1893, it is the county seat of McDowell County. He recently resides in Atlanta, GA and also own properties lived in ........
+                        <hr>
+                    <div class="list">
+                        <ul class="a">
+                            <li>Chicago IL</li>
+                            <li>Pittsburgh PA</li>
+                            <li>Los Angeles</li>
+                            <li>Cleveland OH</li>
+                            <li>Little Elm, TX</li>
+                        </ul>
+                    </div>
+                        <hr>
+                    </p>
+
+                    <p>
+                        Broderick Stephen Harvey, Sr. is an American television host, actor, writer, producer, and comedian. He hosts The Steve Harvey Morning Show, Family Feud, Celebrity Family Feud, the Miss Universe competition, Family Feud Africa, and arbitration-based court comedy, Judge Steve Harvey, and the host of his own successful daytime television talk show there's not much that Steve Harvey hasn't accomplished in his long and illustrious career in Hollywood both on screen and on stage. <a href="https://en.wikipedia.org/wiki/Steve_Harvey"target=" _blank">
+                            Click here to read more....</a>
+                    </p >
+                </div>
+                <div class="cont">
+                    <blockquote>
+                    “Do not ignore your gift. Your gift is the thing you do the absolute best with the least amount of effort.”
+
+                    <br>
+                    <cite>- Steve Harvey</cite>
+                    </blockquote>
+                </div>
+
+    </section>
+
+        </div> 
```

Comment:
Line 56 `div` should be switched with line 54 `section`. It may have worked this time but once websites start getting more complex you'll have issues troubleshooting.

Reply:
Using comments to layout the various pieces of the layout can help make it more readable. Nothing needs to change on this lab, just something to note.